tools/updateMenteeStatus.ts

```diff
@@ -0,0 +1,22 @@
+import sequelize from "../src/api/database/sequelize";
+
+// eslint-disable-next-line @typescript-eslint/no-floating-promises
+main().then();
+async function main() {
+  console.log("Updating mentee status...");
+
+  await sequelize.transaction(async (transaction) => {
```

Comment:
add logic to migrateDatabase.ts and delete this file. Otherwise production won't automatically run it.